core/account-id/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.0.0"
 authors = ["Near Inc <hello@nearprotocol.com>"]
 publish = true
 # Please update rust-toolchain.toml as well when changing version here:
-rust-version = "1.60.0"
+rust-version = "1.64.0"
```

Comment:
This should stay at 1.60 -- `near_account_id` is a crate we publish to crates.io with proper semver API, so we shouldn't be advancing it's rust-version willy-nilly

Reply:
Could you change the comment above instead?